parm/config/gfs/config.wave

```diff
@@ -30,7 +29,6 @@ case "${waveGRD}" in
     export waveGRDN='1 2 3'
     export waveGRDG='10 20 30'
     export USE_WAV_RMP='YES'
-    export waveMULTIGRID='.true.'
```

Comment:
The entire case ""gnh_10m;aoc_9km;gsh_15m")" must be removed as to run it, waveMULTIGRID='.true.'.